handlers/api.go

```diff
@@ -3470,25 +3501,37 @@ func getAuthClaims(r *http.Request) *utils.CustomClaims {
 	return claims.(*utils.CustomClaims)
 }
 
-func returnQueryResults(rows *sql.Rows, w http.ResponseWriter, r *http.Request) {
+func returnQueryResults(rows *sql.Rows, w http.ResponseWriter, r *http.Request, adjustQueryEntriesFuncs ...func(map[string]interface{}) error) {
 	j := json.NewEncoder(w)
 	data, err := utils.SqlRowsToJSON(rows)
 	if err != nil {
 		sendErrorResponse(w, r.URL.String(), "could not parse db results")
 		return
 	}
 
+	err = adjustQueryResults(data, adjustQueryEntriesFuncs...)
+	if err != nil {
+		sendErrorResponse(w, r.URL.String(), "could not parse db results")
```

Comment:
I'm feeling iffy about this error message. Something went wrong while trying to adjust the query entries with the given functions. Does this still count at a "could not parse db results"?

Reply:
No.

```suggestion
		sendErrorResponse(w, r.URL.String(), "could not adjust query results")
```
?

Reply:
Keep in mind that this would have to be changed here too:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/817b4c30dfccb52a0821a6106467af78cd5e0766/handlers/api.go#L3531